templates/prelogin/home.html

```diff
@@ -3,7 +3,7 @@
 {% load hq_shared_tags %}
 
 {% block content %}
-{% initial_page_data 'ab_test' ab_test %}
+{% analytics_ab_test 'kissmetrics.prelogin_video_ab' ab_test %}
```

Comment:
Shouldn't this also update here: https://github.com/dimagi/commcarehq-prelogin/blob/2840a2f8c21a41482591bbdfba74573ab1612fe2/templates/prelogin/base.html#L290 ?

Reply:
yea that can actually be removed